rocksdb_admin/rocksdb_admin.thrift

```diff
@@ -114,6 +114,7 @@ struct AddS3SstFilesToDBRequest {
   2: required string s3_bucket,
   3: required string s3_path,
   4: optional i32 s3_download_limit_mb = 64,
+  5: optional bool create_db = false,
```

Comment:
Can you add an openDB() interface instead?

I'd prefer a set of simple and small interfaces to a smaller set of complicated interfaces.